### ISSUE TITLE
Non-hermetic `basename` tool

### ISSUE THREAD

**Issue by birunts** [GoogleContainerTools/rules_distroless#141]
A `genrule` from [deb_import.bzl](https://github.com/GoogleContainerTools/rules_distroless/blob/main/apt/private/deb_import.bzl#L30) is using non-hermetic `basename` tool. 
This is very simple case here to grab only the file name without any sophisticated usage. Seems to be reasonable to use simple POSIX instead e.g.:

```diff
    # gzip compression.

-    data_file="$$(basename $<)"
+    SRC="$<"
+    data_file="$${SRC##*/}"

    if [[ "$$data_file" == "data.tar.bz2" ]]; then
```

**Comment by jjmaestro** [comment]
Good catch! Looking at it, I inadvertently added it in #99 but it's not really needed, so I've removed it in #143.

I probably left it when I copy-pasted the script from a standalone version that I wrote in my computer, to test a few thing outside of Bazel before moving it into `deb_import` 😅
